src/__mock__/todos.test.js

```diff
@@ -57,3 +57,44 @@ describe('add and remove', () => {
     expect(todoList.list).toHaveLength(2);
   });
 });
+
+describe('Edit test', () => {
+  test('Editing', () => {
+    const todoList = new Todos();
+    const newTodo3 = {
+      id: 'gfgdgrg',
+      description: 'task33',
+      completed: false,
+      index: 3,
+    };
+    todoList.addTodo(newTodo3);
+    todoList.editTodo(newTodo3.id, 'asd');
+    expect(todoList.list[2].description).toBe('asd');
+    expect(todoList.list).toHaveLength(3);
+  });
+});
+
+describe('complete test', () => {
+  test('updating an items completed status', () => {
+    const todoList = new Todos();
+    const newTodo4 = {
+      id: 'dasasds5sa',
+      description: 'task5',
+      completed: false,
+      index: 4,
+    };
+    todoList.addTodo(newTodo4);
+    todoList.completeTodo(newTodo4.id, true);
+    expect(todoList.list[3].completed).toBeTruthy();
+    expect(todoList.list).toHaveLength(4);
+  });
+});
+
+describe('clear all completed', () => {
+  test('clear items completed', () => {
+    const todoList = new Todos();
+    todoList.clearCompletedTodos();
+    expect(todoList.list).toHaveLength(3);
+    expect(todoList.list[1].completed).toBeFalsy();
+  });
+});
```

Comment:
Nice job so far on writing test cases for the `add`, `remove`, `clear completed`, and `edit` functions👍👍 But don't forget we are also required to mock the local storage update and mock the HTML elements to test for DOM functions😉

<img width="626" alt="image" src="https://user-images.githubusercontent.com/64643995/231820915-f9a66512-2e70-496f-8dca-13fe9070450f.png">

Kindly ensure these requirements are met in your project👍.